packages/pluggableWidgets/column-chart-web/src/ColumnChart.editorConfig.ts

```diff
@@ -45,8 +44,13 @@ export function getProperties(
 
         transformGroupsIntoTabs(defaultProperties);
     } else {
-        if (!showAdvancedOptions) {
-            hidePropertiesIn(defaultProperties, values, ["customLayout", "customConfigurations", "enableThemeConfig"]);
+        if (!values.advancedOptions) {
+            hidePropertiesIn(defaultProperties, values, [
+                "customLayout",
+                "customConfigurations",
+                "enableThemeConfig",
+                "developerMode"
+            ]);
```

Comment:
The advanced checks need to be done only in studio (platform = web). The previous editorconfig flow was copy pasted and not entirely correct. [Check example here](https://github.com/mendix/widgets-resources/pull/1350/commits/87f589113b85780aabddb07f395bc03fe819e0cc#diff-80f98e1d2694d49918bd8103babf24954348f8193aad8ddee931b1f9715c20c8L14). The main idea is that the advanced toggle is only available in Studio, while Studio Pro has all options always.